source/extensions/quic_listeners/quiche/platform/envoy_quic_clock.cc

```diff
@@ -9,13 +9,13 @@ quic::QuicTime EnvoyQuicClock::ApproximateNow() const {
 }
 
 quic::QuicTime EnvoyQuicClock::Now() const {
-  return quic::QuicTime::Zero() + quic::QuicTime::Delta::FromMicroseconds(
-                                      microsecondsSinceEpoch(time_system_.monotonicTime()));
+  return quic::QuicTime::Zero() + quic::QuicTime::Delta::FromMicroseconds(microsecondsSinceEpoch(
```

Comment:
This function seems to be not clang-formated, can you do that?

Reply:
It is, otherwise the format ci would have failed.

Reply:
SGTM, thanks for double check.